src/app/modules/backbone/components/header/header.component.html

```diff
@@ -0,0 +1,20 @@
+<nav class="main-nav">
+    <header class="brand">
+        <img src="/favicon.ico"/>
+        <span>DAK</span>
+    </header>
+    <div class="burger-menu-wrapper">
+        <app-burger-menu [onBurgerMenuClick]="onBurgerMenuClick"></app-burger-menu>
+    </div>
+    <div class="nav-spacer"></div>
+    <div class="widget-container">
+        <app-wallet-widget></app-wallet-widget>
+        <app-cart-widget></app-cart-widget>
+    </div>
+    <div class="sidebar-container" [ngClass]="{ 'sidebar-open': showSidebar, 'sidebar-closed': !showSidebar }">
+        <div class="sidebar">
+            <app-sidebar [(open)]="showSidebar"></app-sidebar>
```

Comment:
by importing app-sidebar here, we are creating a second instance of the sidebar component. There is no need to do this. Also, there should be no dependency between the header/sidebar but rather they should communicate through input/output (if we need a service to facilitate, we can also do that)